xbmcgui.py

```diff
@@ -122,6 +122,12 @@ def yesno(self, heading, message, nolabel="No", yeslabel="Yes", customlabel=None
 
         """
 
+        if nolabel is None:
```

Comment:
According to https://romanvm.github.io/Kodistubs/_autosummary/xbmcgui.html#xbmcgui.Dialog it should default to an empty string. So perhaps it should be better to make it like that in line 110 and then here use:

``` 
if not nolabel:
    nolable = "No"
```
That would also catch the official fallback of an empty string?

Reply:
:+1: